libraries/Microsoft.Bot.Builder.AI.QnA/Dialogs/QnAMakerDialog.cs

```diff
@@ -667,7 +694,8 @@ private async Task<DialogTurnResult> DisplayQnAResultAsync(WaterfallStepContext
             // If response is present then show that response, else default answer.
             if (stepContext.Result is List<QueryResult> response && response.Count > 0)
             {
-                await stepContext.Context.SendActivityAsync(response.First().Answer, cancellationToken: cancellationToken).ConfigureAwait(false);
+                var message = QnACardBuilder.GetQnADefaultResponse(response.First(), dialogOptions.ResponseOptions.DisplayPreciseAnswerOnly);
+                await stepContext.Context.SendActivityAsync(message).ConfigureAwait(false);
```

Comment:
>SendActivityAsync(message).ConfigureAwait(false); [](start = 42, length = 49)

Send cancellation token.

Reply:
Implemented the suggestion.

Reply:
Modified as per suggestion.

---
In reply to: [433511382](https://github.com/microsoft/botbuilder-dotnet/pull/3935#discussion_r433511382) [](ancestors = 433511382)